### PR TITLE
Add documentation for the new `migrations mark_migrated` command

### LINKED ISSUE #2646: Document the new `migrations mark_migrated` command

### PR DESCRIPTION
**Pull request by HavokInspiration**
Add documentation for the new `migrations mark_migrated` command

Closes #2646

### CODE REVIEW

File: en/migrations.rst [hunk 1]
```diff
@@ -262,6 +262,20 @@ You can use this command to determine which migrations have been run::
 
         bin/cake migrations status
 
+Marking a migration as migrated
+=================
```
Comment (antograssiot) [inline en/migrations.rst]: mist be at least as long as the line above

File: fr/migrations.rst [hunk 2]
```diff
@@ -283,6 +283,20 @@ migrations qui ont été exécutées::
 
         bin/cake migrations status
 
+Marqué une migration comme "migrée"
+=====================
```
Comment (lorenzo) [inline fr/migrations.rst]: Same here

**Comment by HavokInspiration** [comment]
I will make the modifications.
Probably on Wednesday, I doubt I'll have the time to do it today.

**Comment by HavokInspiration** [comment]
Changes made and PR rebased.

**Comment by antograssiot** [comment]
thank's @HavokInspiration

**Pull request merged**